<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowMain</class>
 <widget class="QWidget" name="WindowMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="WindowMainLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QGraphicsView" name="GraphicsView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>99</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="renderHints">
      <set>QPainter::Antialiasing</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="MainMenu">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>85</height>
      </size>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="TabElectronicComponents">
      <attribute name="title">
       <string>Logic components</string>
      </attribute>
      <widget class="QPushButton" name="LogicSource">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>60</width>
         <height>60</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>Source</string>
       </property>
      </widget>
      <widget class="QPushButton" name="LogicTautology1">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>0</y>
         <width>60</width>
         <height>60</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>YES</string>
       </property>
      </widget>
      <widget class="QPushButton" name="LogicTautology2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>0</y>
         <width>60</width>
         <height>60</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>YES x2</string>
       </property>
      </widget>
      <widget class="QPushButton" name="LogicTautology3">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>0</y>
         <width>60</width>
         <height>60</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>YES x3</string>
       </property>
      </widget>
      <widget class="QPushButton" name="LogicNegation">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>0</y>
         <width>60</width>
         <height>60</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>NOT</string>
       </property>
      </widget>
      <widget class="QPushButton" name="LogicConjunction">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>0</y>
         <width>60</width>
         <height>60</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>AND</string>
       </property>
      </widget>
      <widget class="QPushButton" name="LogicDisjunction">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>0</y>
         <width>60</width>
         <height>60</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>OR</string>
       </property>
      </widget>
      <widget class="QPushButton" name="LogicImplication">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>0</y>
         <width>60</width>
         <height>60</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>→</string>
       </property>
      </widget>
      <widget class="QPushButton" name="LogicBiconditional">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>0</y>
         <width>60</width>
         <height>60</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>↔</string>
       </property>
      </widget>
      <widget class="QPushButton" name="LogicIndicator">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>0</y>
         <width>60</width>
         <height>60</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>Indicator</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="TabOperations">
      <attribute name="title">
       <string>Operations</string>
      </attribute>
      <widget class="QPushButton" name="OperationClearAll">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>60</width>
         <height>60</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>Clear all</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="TabLanguages">
      <attribute name="title">
       <string>Language</string>
      </attribute>
      <widget class="QPushButton" name="SetLanguageEnglish">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>60</width>
         <height>60</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>English</string>
       </property>
      </widget>
      <widget class="QPushButton" name="SetLanguageRussian">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>0</y>
         <width>60</width>
         <height>60</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>Русский</string>
       </property>
      </widget>
      <widget class="QPushButton" name="SetLanguageDeutsch">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>0</y>
         <width>60</width>
         <height>60</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>Deutsch</string>
       </property>
      </widget>
      <widget class="QPushButton" name="SetLanguageFrench">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>0</y>
         <width>60</width>
         <height>60</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>Français</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="TabInformation">
      <attribute name="title">
       <string>Information</string>
      </attribute>
      <widget class="QPushButton" name="InformationAbout">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>60</width>
         <height>60</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>About</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
